eclipse-scout-core/src/layout/logicalgrid/GridData.ts

```diff
@@ -168,6 +168,13 @@ export class GridData {
     $.extend(this, model);
   }
 
+  equals?(other: GridData) {
```

Comment:
Until now, the GridData object contained no methods and could be used as model object. To prevent compile errors I added the ? modifier.

I agree, it is confusing. We should probably create a GridDataModel instead